### PR TITLE
Sealed Decklist Basic Draft

### PR DESCRIPTION
**Pull request by tpblaster**
This is a basic implementation of the code to add decklists to sealed products. This is currently a draft and has a lot of issues.

### CODE REVIEW

File: mtgjson5/providers/tcgplayer.py [hunk 1]
```diff
@@ -251,7 +251,7 @@ def generate_mtgjson_sealed_product_objects(
 
         for product in sealed_data:
             sealed_product = MtgjsonSealedProductObject()
-            sealed_product.name = product["cleanName"]
+            sealed_product.name = product["name"]
```
Comment (kodawah) [inline mtgjson5/providers/tcgplayer.py]: why is this needed?

**Comment by kodawah** [comment]
could you post an example output from this code?

**Comment by tpblaster** [comment]
Here is the output for War of the Spark
[WAR.zip](https://github.com/mtgjson/mtgjson/files/7919357/WAR.zip)

**Comment by kodawah** [comment]
@tpblaster I was hoping to check the output of a decklist-based set, like a commander edition which has multiple products containing different cards from the same set
